bin/run_unittests.sh

```diff
@@ -107,6 +107,8 @@ function run_libclipper_tests {
 function run_management_tests {
   echo -e "\nRunning management tests\n\n"
   ./src/management/managementtests --redis_port $REDIS_PORT
+  cd $DIR
+  python ../management/test/clipper_manager_test.py
```

Comment:
`clipper_manager.py` got moved from the `management/` to the `clipper_admin/` directory as part of making the pip package. Move the tests to the same place (`clipper_admin/test`).